jdaviz/configs/mosviz/helper.py

```diff
@@ -288,31 +290,32 @@ def _row_click_message_handler(self, msg):
     def _handle_image_zoom(self, msg):
         mos_data = self.app.data_collection['MOS Table']
 
-        # trigger zooming the image, if there is an image
-        if mos_data.find_component_id("Images") is not None:
-            if msg.shared_image:
-                center, height = self._zoom_to_object_params(msg)
-            else:
-                try:
-                    center, height = self._zoom_to_slit_params(msg)
-                except IndexError:
-                    # If there's nothing in the spectrum2d viewer, we can't get slit info
-                    return
```

Comment:
I don't see an equivalent to this catch. Was it a conscious decision to un-handle this exception?

Reply:
This is now caught within `_zoom_to_slit_params`. If somehow we still get an error here, it is a bug and we should just let the exception raise naturally.